chain/cosmos/genesis.go

```diff
@@ -0,0 +1,48 @@
+package cosmos
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/icza/dyno"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+)
+
+type GenesisKV struct {
+	Key   string      `json:"key"`
+	Value interface{} `json:"val"`
+}
+
+func ModifyGenesis(genesisKV []GenesisKV) func(ibc.ChainConfig, []byte) ([]byte, error) {
```

Comment:
Huge fan of the generic way you implemented this 👏 

It beats the v1beta1 vs v1 check we are doing in the current `ModifyGenesisProposalTime` function.

Would you be able to bring this up to date with `main` use your `ModifyGenesis` instead of the current `ModifyGenesisProposalTime`

Reply:
:pray: Thank you! Resolved in 2b9025f84e739cfc0979766b093af37c00de77ca when I merged main in